components/tooltip.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Here is the key part. We just wrap the React Bootstrap component and received the minimun amount of properties to satisfy used properties in the rest of the code.

Take into consideration that `style` and `placement` are used only in 1 place, and `positionLeft` is only used in other 2 places, probably the component should be better if we just make that 2 cases something like the `type` of tooltip, or something like that.

Reply:
@jespino should we add a few simple test cases as well

Reply:
I can create an snapshot test, but because this is just a wrapper, is not going to get much advantage from testing for now. In the future, if we change the API or the logic to convert from outside properties to the underlaying properties, probably makes more sense, for now I would leave it without tests. But I'm happy to add them if you see value in it.